Detector/DetFCChhTrackerTkLayout/src/BeamTube_geo.cpp

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
Why can't we move it up one additional layer to DetCommon? Is there any dependency on the FCChh geometry (except from dimensions)?